Sources/MUXSDKStats/MUXSDKStats.m

```diff
@@ -523,12 +523,6 @@ + (void)programChangeForPlayer:(nonnull NSString *)name
     }
 }
 
-+ (void)programChangeForPlayer:(nonnull NSString *)name
```

Comment:
We should do a major rev to remove this from the public API

Would you consider deprecating it instead?